### ISSUE TITLE
Flash Backbone

### ISSUE THREAD

**Issue by aribornstein** [nateraw/huggingface-vit-finetune#1]
This is cool it could make a great backbone integration for Flash. I'll mark it for later.

**Comment by nateraw** [comment]
Great idea! Yeah let's wait until this feature gets ironed out a bit more by their team. Probably masochism to implement it in flash til then.

**Comment by motoight** [comment]
Hi there, I am doing ViT finetuning on CIFAR10 too! And I want to know what classification accuracy did you fine-tuned model get.
Thank you for share your findings!
